include/quda.h

```diff
@@ -1727,7 +1727,7 @@ extern "C" {
    * @param[in,out] obs_param Parameter struct that defines which
    * observables we are making and the resulting observables.
    */
-  void performAdjGFlowHier(void *h_out, void *h_in, QudaInvertParam *inv_param, QudaGaugeSmearParam *smear_param);
+  void performAdjGFlowHier(void **h_out, void **h_in, QudaInvertParam *inv_param, QudaGaugeSmearParam *smear_param, const size_t nSpinors);
```

Comment:
Same as above: need to make the doxygen consistent with the function and remove `const` from `nSpinors`.